core/src/main/scala/cats/syntax/option.scala

```diff
@@ -27,6 +27,17 @@ final class OptionIdOps[A](val a: A) extends AnyVal {
 }
 
 final class OptionOps[A](val oa: Option[A]) extends AnyVal {
+  /**
+    * If the `Option` is `Some` return its value in a `scala.util.Right`.
+    * If the `Option` is `None`, return the provided `B` value in a `scala.util.Left`.
+    *
+    * @see [[cats.syntax.EitherObjectOps#fromOption]]
+    */
+  def toEither[B](ifNone: => B): Either[B, A] = oa match {
```

Comment:
What is the benefit over `.toRight` provided by the standard library?

Reply:
Doh! what a fantastic point. I can't believe I forgot about that. Thanks!